lib/constants/stage-operators.js

```diff
@@ -246,7 +246,7 @@ const STAGE_OPERATORS = [
     score: 1,
     meta: 'stage',
     version: '2.2.0',
-    description: 'Skips the first n documents where n is the specified skip number and passes the remaining documents unmodified to the pipeline.',
+    description: 'Skip a specified number of documents before advancing to the next stage.',
```

Comment:
Skip or skips? I have noticed here you were changing verbs to form with `s`.

Reply:
Thanks @alenakhineika! updated.